mist-core/src/main/java/edu/snu/mist/core/parameters/OverloadedTaskThreshold.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2018 Seoul National University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.snu.mist.core.parameters;
+
+import org.apache.reef.tang.annotations.Name;
+import org.apache.reef.tang.annotations.NamedParameter;
+
+/**
+ * The threshold for determining whether a task is overloaded or not.
+ */
+@NamedParameter(doc = "The overloaded task threshold.", default_value = "0.95", short_name = "over_thred")
```

Comment:
Maybe we need to consider that using only CPU load is enough to represent Task's load.

Reply:
Created on #1013.